src/rpc/masternode.cpp

```diff
@@ -227,7 +227,8 @@ UniValue masternode_count(const JSONRPCRequest& request)
     int nCount;
     int total;
     if (deterministicMNManager->IsDeterministicMNsSporkActive()) {
-        nCount = total = mnodeman.CountEnabled();
+        total = mnodeman.CountMasternodes();
```

Comment:
Same as above but could unify `total` for both cases now (and make it consistent with the way it's calculated in `ClientModel::getMasternodeCountString()`) I guess i.e.
```diff
diff --git a/src/rpc/masternode.cpp b/src/rpc/masternode.cpp
index 18255a559..f443a0527 100644
--- a/src/rpc/masternode.cpp
+++ b/src/rpc/masternode.cpp
@@ -225,14 +225,12 @@ UniValue masternode_count(const JSONRPCRequest& request)
         masternode_count_help();
 
     int nCount;
-    int total;
+    int total = mnodeman.CountMasternodes(0);
     if (deterministicMNManager->IsDeterministicMNsSporkActive()) {
-        total = mnodeman.CountMasternodes();
         nCount = mnodeman.CountEnabled();
     } else {
         masternode_info_t mnInfo;
         mnodeman.GetNextMasternodeInQueueForPayment(true, nCount, mnInfo);
-        total = mnodeman.size();
     }
 
     int ps = mnodeman.CountEnabled(MIN_PRIVATESEND_PEER_PROTO_VERSION);
```

Reply:
Applied suggestion :+1: